Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/FileTransporter.cs

```diff
@@ -5,461 +5,472 @@
 using System.Text;
```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/54829047d8be4e96cb94fdb7ef839a418b4b3c7e/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/FileTransporter.cs#L3
You are not using this namespace, so it can be removed.

Reply:
Removed